metron-interface/metron-config/scripts/package.json

```diff
@@ -9,7 +9,8 @@
     "http-proxy-middleware": "0.17.4",
     "optimist": "0.6.1",
     "serve-favicon": "2.4.2",
```

Comment:
I think a license check for npm dependencies would be a good thing and we should include it. 
 
Should it be required for this PR?  @mattf-horton is preparing the next release and this is the last PR he's waiting on.  If this needs to be included he should probably proceed without it because I don't think this will be a quick or easy change.